eap-aro/src/main/bicep/mainTemplate.bicep

```diff
@@ -157,7 +156,7 @@ resource uami 'Microsoft.ManagedIdentity/userAssignedIdentities@${azure.apiVersi
 
 // Assign Contributor role in subscription scope since we need the permission to get/update resource cross resource group.
 module deploymentScriptUAMICotibutorRoleAssignment 'modules/_rolesAssignment/_roleAssignmentinSubscription.bicep' = {
-  name: name_roleAssignmentName
+  name: const_roleAssignmentName
```

Comment:
Looking at line 152

```
resource uami 'Microsoft.ManagedIdentity/userAssignedIdentities@${azure.apiVersionForIdentity}' existing = {
```

I am reminded that we have not updated the apiVersions in `utilities/azure-common.properties` in a long time.

I am worried that when we go to publish this offer it will fail the ARM TTK due to some of the apiVersions being too old.

@backwind1233 , can you please add to this PR the work of updating the apiVersions to the latest possible versions?